terraform/variables.tf

```diff
@@ -1,13 +1,19 @@
 variable "image_path" {
   type        = string
   description = "Where the images will be stored"
-  default     = "/code/ubuntu-pool"
+  default     = "/home/user/images"
+}
+
+variable "private_key_path" {
+  type        = string
+  description = "SSH private key path"
+  default     = "/home/user/.ssh/k8s_rsa"
 }
 
 variable "ssh_key" {
   type        = string
   description = "SSH pub key to use"
-  default     = "<contents of ~/.ssh/id_rsa.pub>"
+  default     = "ssh-rsa ..."
```

Comment:
Same here - just remove it. It will prompt the user to fill in the value

Reply:
ok 👍